nilearn/decomposition/tests/test_canica.py

```diff
@@ -3,17 +3,34 @@
 import numpy as np
 from numpy.testing import assert_array_almost_equal
 from nose.tools import assert_true, assert_raises
-
 import nibabel
-
 from nilearn.decomposition.canica import CanICA
 
 
-def test_canica_square_img():
-    shape = (20, 20, 1)
-    affine = np.eye(4)
-    rng = np.random.RandomState(0)
+try:
```

Comment:
cf. `assert_raises_regex` appears to be similar case and is located in nilearn/_utils/testing.py